function/templates/_servicebus-trigger.yaml

```
@@ -9,6 +9,6 @@
     topicName: {{ .topicName }}
     subscriptionName: {{ .subscriptionName }}
     {{- end }}
-    connection: {{ .connection }}
-    queueLength: {{ .queueLength | quote }}
+    connectionFromEnv: {{ .connection }}
+    messageCount: {{ .queueLength | quote }}
```

Should we rename these and make a breaking change so its easy to refer Keda docs ?
```suggestion
    connectionFromEnv: {{ .connectionFromEnv }}
    messageCount: {{ .messageCount | quote }}
```

---

how about `connectionFromEnv: {{ coalesce .connectionFromEnv .connection }}` to avoid the BC?

---

perfect, assuming it gives preference to connectionFromEnv if both present .